Clockwise/VirtualClock.cs

```diff
@@ -74,8 +73,7 @@ private VirtualClock(DateTimeOffset? now = null, [CallerMemberName] string creat
             }
         }
 
-        public async Task AdvanceBy(TimeSpan timespan) =>
-            await AdvanceTo(now.Add(timespan));
+        public Task AdvanceBy(TimeSpan timespan) => AdvanceTo(now.Add(timespan));
```

Comment:
No need to make this function `async`, just return the task that was provided by `AdvanceTo`.